dwave/system/temperatures.py

```diff
@@ -249,373 +384,685 @@ def maximum_pseudolikelihood_temperature(bqm=None,
+        en1: Effective fields as an np.ndarray (site labels not required).
+            Derived from the ``bqms`` and ``sampleset`` if not provided.
+            First dimension indexes samples, second dimension indexes sites.
+            Ordering doesn't matter, but should be consistent with sample_weights.
+        num_bootstrap_samples: Number of bootstrap estimators to calculate.
+            Bootstrapped estimates can be used to reliably estimate variance and
+            bias if samples are uncorrelated. For
+            now, the sampleset must have uniform sample_weights to deploy this
+            option -- an aggregated or weighted sampleset must be disaggregated
+            (raw format) with repetitions.
+        seed: Seeds the bootstrap method (if provided) allowing reproducibility
+            of the estimators.
+        x0: Initial guess for the fitting parameters. Should have the same
+            length as bqms when provided.
+        optimize_method (str,optional, default=None):
+            Optimize method used by SciPy ``root_scalar`` method. The default
+            method works well under default operation, 'bisect' can be
+            numerically more stable for the scalar case (Temperature estimation
+            only).
+        bisect_bracket: Relevant only if optimize_method='bisect' and for a
+            single bqm. Bounds the fitting parameter.
+        sample_weights: A set of weights for the samples. If sampleset is of
+            type :obj:`~dimod.SampleSet` set this is default to
+            sampleset.record.num_occurrences, otherwise uniform weighting is
+            the default.
+        return_optimize_object: When True, and if ``sciPy`` optimization is
+            invoked, the associated OptimizeResult is returned. Otherwise only
+            the optimal parameter values are returned as floats.
+        degenerate_fields: If effective fields are degenerate owing to sparse
+            connectivity, low precision and/or large number of samples, or low
+            entropy then histogramming (aggregating) of fields is used to
+            accelerate the search stage. A value True is supported (and default)
+            for single parameter esimation ``len(bqms)=1``; for multi-parameter
+            estimation only value False is supported.
+        use_jacobian: By default (True) the second derivative of the
+            pseudolikelihood is calculated and used by ScipY root finding
+            methods. The associated complexity of this non-essential
+            calculation is quadratic in len(bqms); use of the second derivative
+            is disabled by setting the value to False.
+    Returns:
+        Tuple of the optimal parameters and a List of bootstrapped estimators:
+            (x_estimate, x_bootstrap_estimates)
+
+            *x_estimate*: a temperature estimate
+            *x_bootstrap_estimates*: a numpy array of bootstrap estimators
```

Comment:
This renders better if written as a block, with all lines on the same indentation level:
```
Returns:
    Tuple: The optimal parameters and a list of bootstrapped estimators (x_estimate, x_bootstrap_estimates):

    * *T_estimate*: a temperature estimate
    * *T_bootstrap_estimates*: a numpy array of bootstrap estimators
```
Same for `maximum_pseudolikelihood_temperature` above.